autogpt/prompts/prompt.py

```diff
@@ -38,7 +38,7 @@ def build_default_prompt_generator() -> PromptGenerator:
     )
     prompt_generator.add_constraint("No user assistance")
     prompt_generator.add_constraint(
-        'Exclusively use the commands listed in double quotes e.g. "command name"'
```

Comment:
Upload cassettes (or I can)

Reply:
I'm happy to  but not sure how?
Can you point me in a direction?

Reply:
Done.